tests/test_E2_economics.py

```diff
@@ -86,50 +123,115 @@
 
 def test_all_cost_info_parameters_added_to_dict_asset():
     """Tests whether the function get_costs is adding all the calculated costs to dict_asset."""
+    print(dict_economic)
```

Comment:
to be removed later

Reply:
Removed in #525, 5e4ff87352d2012f5e49973b13047cb327d1ffd7